### PR TITLE
Implement the X-Elastic-Client-Meta HTTP header

### PR DESCRIPTION
**Pull request by Anaethelion**

### CODE REVIEW

File: estransport/estransport.go [hunk 1]
```diff
@@ -194,6 +200,11 @@ func New(cfg Config) (*Client, error) {
 // Perform executes the request and returns a response or error.
 //
 func (c *Client) Perform(req *http.Request) (*http.Response, error) {
+	_, file, no, ok := runtime.Caller(-1)
```
Comment (Mpdreamz) [inline estransport/estransport.go:203]: I'm sure this was considered so asking to learn, whats the performance cost of this in Go?

Comment (Anaethelion) [reply estransport/estransport.go:203]: Good catch, that was not supposed to be committed !

File: esutil/bulk_indexer_internal_test.go [hunk 2]
```diff
@@ -629,6 +630,111 @@ func TestBulkIndexer(t *testing.T) {
...
+			name string
+			args args
+			want string
+		}{
+			{
+				name: "Meta header is present in header",
+				args: args{
+					disableMetaHeader: false,
+				},
+				want: `.*,h=bp`,
```
Comment (sethmlarson) [inline esutil/bulk_indexer_internal_test.go:649]: Just verifying that there won't be a space between `.*` and `,`? I don't know how Go's newline folding works by default :)

Comment (Anaethelion) [reply esutil/bulk_indexer_internal_test.go:649]: There shouldn't be, but I'll modify the test to exclude this possibility!

**Review by sethmlarson — APPROVED** [review]
Implementation seems fine to me :+1:

**Pull request merged**